### DISCUSSION TITLE
how to remove the ability to load from the gallery and draw the grid in the application when starting the camera

### DISCUSSION THREAD

**Discussion by PachinoMax** [quasarframework/quasar#9686] [CLI - Capacitor / Cordova modes]
Please tell me how to remove the ability to load from the gallery and draw the grid in the application when starting the camera? The task is to identify the client by the document, so that he cannot upload photos from the gallery, it is necessary to remove the buttons from the screen so that the user does not have such an opportunity. And the grid so that he photographs the document in this area.
And the ability to switch the camera from front to back how can you help
I use capactitor.

**Comment by metalsadman** [comment]
look for a camera plugin that does it for you whether cordova one's that are compatible for capacitor.

**Reply by ontwikkelfabriek** [reply]
@PachinoMax 

> Please tell me how to remove the ability to load from the gallery 

You can do that with the cordova camera plugin, with `Camera.PictureSourceType.CAMERA`:
https://cordova.apache.org/docs/en/10.x/reference/cordova-plugin-camera/#camerapicturesourcetype--enum
